src/main/java/org/zaproxy/zap/extension/jwt/fuzzer/messagelocations/FuzzerJWTSignatureOperation.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2020 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.jwt.fuzzer.messagelocations;
+
+import org.zaproxy.zap.extension.jwt.JWTI18n;
+
+/**
+ * This class represents the operation on JWT Signature after fuzzing the {@code Headers} and {@code
+ * Payload} fields. <br>
+ * Following are the operations on the Signatures:
+ *
+ * <ol>
+ *   <li>Either no signature component will be added to JWT token e.g. in case of {@code None}
```

Comment:
I'm probably over thinking this but saying "JWT Token" is like saying "token token" because the T in the acronym is 'Token'...

Reply:
This comment applies elsewhere as well.

Reply:
hmm correct...  will change those all.